src/components/shift-schedule/RegisterSection.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from '@/components/ui/Button';
+import { Checkbox } from '@/components/ui/Checkbox';
+import { Label } from '@/components/ui/Label';
+import { cx } from '@/lib/utils';
+import { useTranslations } from 'next-intl';
+
+function RegisterSection({ className }: { className?: string }) {
+  const t = useTranslations(
+    'shiftSchedule.scheduleTable.scheduleCellDialog.registerSection',
+  );
+
+  return (
+    <section className={cx(className, 'space-y-3')}>
```

Comment:
Same as before the use of the section element is not semantically correct. Maybe also renaming the component itself can be smart to avoid confusion